.golangci.yml

```diff
@@ -1,3 +1,5 @@
+run:
+  timeout: 5m
```

Comment:
could it be you need to rebase as this one should be there?

Reply:
Definitely. Haven't contributed to a GitHub project before, so I'm still learning the ropes.